src/create_discretized_sphere.cpp

```diff
@@ -0,0 +1,109 @@
+#include<ros/ros.h>
+#include "reuleaux/PointArray.h"
+#include "geometry_msgs/Point.h"
+#include<octomap/octomap.h>
+#include<octomap/MapCollection.h>
+#include <octomap/math/Utils.h>
+#include <reuleaux/sphere_discretization.h>
+using namespace octomap;
+using namespace std;
+using namespace octomath;
+using namespace sphere_discretization;
+
+
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "rand_points");
+    ros::NodeHandle n;
+  
+    ros::Publisher point_pub = n.advertise<reuleaux::PointArray>("points", 1000);
+
+  //ros::Rate loop_rate(10);
+  ros::Rate poll_rate(100);
+while(point_pub.getNumSubscribers() == 0)
```

Comment:
while(point_pub.getNumSubscribers() == 0) is included as when the generated data is quite big in size, the message sending process takes a lot of time, so if the subscriber(the visualizer) is not subscribed at that time, the map will not be seen in rviz. Then the subscriber is connected, at that time only the message gets published.